ucuapi/protocols/dummy/module.py

```diff
@@ -44,9 +48,11 @@ def dummy_device(udid):
         if dev.udid == udid:
             device = {}
             device["udid"] = dev.udid
+            device["devType"] = dev.devType
             device["values"] = dev.values
             data.append(device)
-    return json.dumps(data)
```

Comment:
This looks like it changes the endpoint to only return one device. Can you confirm that it still returns all of them?

Reply:
This change is under the /device/<udid> route so it's supposed to return one device. I just added a 404 if the device does not exist, instead of an empty json form.

Reply:
@aim17 Ah, my bad. I misread and thought it was on the /devices endpoint.